.teamcity/settings.kts

```diff
@@ -37,275 +37,284 @@ version = "2020.1"
-        root(WpCalypso)
-
-        cleanCheckout = true
-        branchFilter = """
-            +:*
-            +:wip/docker-for-ci
-        """.trimIndent()
-        excludeDefaultBranchChanges = true
-    }
-
-    steps {
-        script {
-            name = "Build docker images"
-            scriptContent = """
-                VERSION="%build.number%"
-                BUILDER_IMAGE_NAME="registry.a8c.com/calypso/base"
-                CI_IMAGE_NAME="registry.a8c.com/calypso/ci"
-                BUILDER_IMAGE="${'$'}{BUILDER_IMAGE_NAME}:${'$'}{VERSION}"
-                CI_IMAGE="${'$'}{CI_IMAGE_NAME}:${'$'}{VERSION}"
-                
-                docker build -f docker/Dockerfile --no-cache --target builder -t "${'$'}BUILDER_IMAGE" .
-                docker build -f docker/Dockerfile --target ci -t "${'$'}CI_IMAGE" .
-                
-                docker tag "${'$'}BUILDER_IMAGE" "${'$'}{BUILDER_IMAGE_NAME}:latest"
-                docker tag "${'$'}CI_IMAGE" "${'$'}{CI_IMAGE_NAME}:latest"
-                
-                docker push "${'$'}CI_IMAGE"
-                docker push "${'$'}{CI_IMAGE_NAME}:latest"
-                
-                docker push "${'$'}BUILDER_IMAGE"
-                docker push "${'$'}{BUILDER_IMAGE_NAME}:latest"
-            """.trimIndent()
-            dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
-            dockerRunParameters = "-u %env.UID%"
-        }
-    }
-
-    features {
-        perfmon {
-        }
-        dockerSupport {
-            loginToRegistry = on {
-                dockerRegistryId = "PROJECT_EXT_6"
-            }
-        }
-    }
+	name = "Build base images"
+	description = "Build base docker images"
+
+	buildNumberPattern = "%build.prefix%.%build.counter%"
+
+	params {
+		param("build.prefix", "1.0")
+	}
+
+	vcs {
+		root(WpCalypso)
+
+		cleanCheckout = true
+		branchFilter = """
+			+:master
+		""".trimIndent()
+		excludeDefaultBranchChanges = true
+	}
+
+	steps {
+		script {
+			name = "Build docker images"
+			scriptContent = """
+				VERSION="%build.number%"
+				BUILDER_IMAGE_NAME="registry.a8c.com/calypso/base"
+				CI_IMAGE_NAME="registry.a8c.com/calypso/ci"
+				BUILDER_IMAGE="${'$'}{BUILDER_IMAGE_NAME}:${'$'}{VERSION}"
+				CI_IMAGE="${'$'}{CI_IMAGE_NAME}:${'$'}{VERSION}"
+
+				docker build -f Dockerfile.base --no-cache --target builder -t "${'$'}BUILDER_IMAGE" .
+				docker build -f Dockerfile.base --target ci -t "${'$'}CI_IMAGE" .
+
+				docker tag "${'$'}BUILDER_IMAGE" "${'$'}{BUILDER_IMAGE_NAME}:latest"
+				docker tag "${'$'}CI_IMAGE" "${'$'}{CI_IMAGE_NAME}:latest"
+
+				docker push "${'$'}CI_IMAGE"
+				docker push "${'$'}{CI_IMAGE_NAME}:latest"
+
+				docker push "${'$'}BUILDER_IMAGE"
+				docker push "${'$'}{BUILDER_IMAGE_NAME}:latest"
+			""".trimIndent()
+			dockerImagePlatform = ScriptBuildStep.ImagePlatform.Linux
+			dockerRunParameters = "-u %env.UID%"
+		}
+	}
+
+	triggers{
+		schedule {
+			schedulingPolicy = daily {
+				hour = 0
```

Comment:
(Config generated using TeamCity UI)

According to the doc, this is 00:00 UTC. It will run the build every 24h, meaning that the image `calypso/ci:latest` will always contain a recent cache